layouts/index.html

```diff
@@ -8,10 +8,6 @@
 </article>
 {{ end }}
 
-<div class="final-message-font center mt2 tc f4 f5-ns fw4 pt4-ns ph5-ns lh-title" style="color:{{ $.Param "final_message_color" | default "#181818" }};">
-    {{ .Params.final_message }}
-</div>
-
```

Comment:
Fix this space

![image](https://github.com/user-attachments/assets/7f9f2ecd-56c9-4072-8ae7-09d61238ee1b)